docs/get_started.rst

```diff
@@ -1,17 +1,388 @@
+| Test Name Match                                                  | The CumulusCI Apex test runner uses a SOQL ``WHERE`` clause to select which tests to run.  Enter the SOQL pattern to match test class names.                                                                                                                                                          |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Do you want to check Apex code coverage when tests are run?      | If yes, checks Apex code coverage when tests are run.                                                                                                                                                                                                                                                 |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Minimum code coverage percentage                                 | Sets the minimum allowed code coverage percentage for your project.                                                                                                                                                                                                                                   |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+
+Verify Project Initialization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Verify project initialization was successful by verifying that ``cumulusci.yml`` exists and has contents.
+
+.. code-block:: console
+
+    $ cat cumulusci.yml
+    project:
+        name: SampleProjectName
+        package:
+            name: SamplePackageName
+            namespace: sampleNamespace
+        .
+        .
+        .
+
+The ``cumulusci.yml`` file configures your project-specific tasks, flows, and customizations.
+
+For more information regarding configuration, check out our `project configuration <#TODO internal ref here>`_ section of the docs.
```

Comment:
Is this cross reference really needed here? It seems to interrupt the description of the cumulusci.yml file, and the "You can even" bit that follows. Delete if possible. If it is needed, consider stringing the sentence immediately after the preceding one, rather than making it its own paragraph. On first reading, I didn't realize they were so related.

Reply:
I think this could be removed. If users are really itching to look at how to configure things I'm hoping that the section title "Configuring CumulusCI" will be obvious enough.